components/server/src/orgs/organization-service.ts

```diff
@@ -363,7 +365,10 @@ export class OrganizationService {
 
     async getSettings(userId: string, orgId: string): Promise<OrganizationSettings> {
         await this.auth.checkPermissionOnOrganization(userId, "read_settings", orgId);
-        const settings = await this.teamDB.findOrgSettings(orgId);
+        const settings = (await this.teamDB.findOrgSettings(orgId)) ?? {};
+        if (!settings.defaultWorkspaceImage) {
```

Comment:
why is it necessary now? we are changing the implementaiton contract, before empty string was default, but now how do we know?

What about `updateSettings`?

Reply:
To be clear we can change it yet, but then we should check on API surface that we consistent, i.e. it should go in `toSettings` here

Secondly we should verify tests and other paths on the dashboard that they work like before, i.e. updating default workspace image settings for both JSON-RPC shim (given old contract there, it should stay backward compatible) and gRPC directly.